lib/sdf/link.rb

```diff
@@ -18,5 +18,39 @@ def each_sensor
         def pose
             Conversions.pose_to_eigen(xml.elements["pose"])
         end
+
+        # The link's inertial.
+        #
+        # @return Inertial struct
+        Inertial = Struct.new(:pose, :mass, :inertia)
+        def inertial
+            mass = read_float_child_element(xml, "mass", 1)
+            ixx = read_float_child_element(xml, "ixx", 1)
+            iyy = read_float_child_element(xml, "iyy", 1)
+            izz = read_float_child_element(xml, "izz", 1)
+            ixy = read_float_child_element(xml, "ixy", 0)
+            ixz = read_float_child_element(xml, "ixz", 0)
+            iyz = read_float_child_element(xml, "iyz", 0)
+            pose = Eigen::Isometry3.new
+             if inertial_elements = xml.elements["inertial"]
```

Comment:
I just realize the [test_link.rb](https://github.com/rock-control/control-ruby_sdformat/pull/13/commits/fcef935a8ccedfb7658ff5fe5a848893ff6bf7a3#diff-9fd23da99c193f6dbc4be8ec5ae74f74R28) is wrong and failing. I'll fix it